ReactAndroid/src/main/java/com/facebook/react/views/scroll/ReactHorizontalScrollView.java

```diff
@@ -92,6 +92,10 @@ public void setPagingEnabled(boolean pagingEnabled) {
     mPagingEnabled = pagingEnabled;
   }
 
+  public boolean flashScrollIndicators() {
```

Comment:
What is the boolean returned by this? Since we're not using it might as well not return it.